sycl/doc/design/OptionalDeviceFeatures.md

```diff
@@ -520,6 +520,10 @@ We add a new IR phase to the device compiler which does the following:
   graph so that each function lists the aspects used by that function and by
   any functions it calls.
 
+* If a function has the `!sycl_declared_aspects` metadata, creates (or augments)
+  the function's `!sycl_used_aspects` metadata with aspects from the
+  `!sycl_declared_aspects` list.
```

Comment:
Do you mind moving this bullet up, so that it is the second bullet in the list, making the order:

* Creates (or augments) ...
* If a function has the `!sycl_declared_aspects` ...
* Propagates each function's `!sycl_used_aspects` ...

This will make it extra clear that the aspects added in step 2 are also propagated in step 3.